app/pages/terms/terms.js

```diff
@@ -100,18 +107,19 @@ var TermsOverlay = React.createClass({
 
     return (
       <div ref='acceptTermsStep'>
-        <div className='terms-overlay-title'>TERMS OF USE</div>
+        <div className='terms-title'>TERMS OF USE</div>
         {terms}
-        <div className='privacy-overlay-title'>PRIVACY POLICY</div>
+        <div className='privacy-title'>PRIVACY POLICY</div>
         {privacy}
-        <form className='terms-overlay-form'>
-          {agreeConfirmation}
-          {backBtn}
-          <button
-            className='terms-button terms-button-submit'
-            onClick={this.handleTermsAndPrivacySubmit}
-            disabled={continueBtnDisabled}>Continue</button>
-        </form>
+        {this.props.authenticated && !this.props.termsAccepted ?
+          <form className='terms-form'>
+            {agreeConfirmation}
+            {backBtn}
+            <button
+              className='terms-button terms-button-submit'
+              onClick={this.handleTermsAndPrivacySubmit}
+              disabled={continueBtnDisabled}>Continue</button>
+          </form> : '' }
```

Comment:
Style comment here - we like to create a var to store contents like this rather than have the ternary operator buried in the JSX where it's a bit hard to read.